apps/builder/src/hooks/useUndoShortcut.ts

```diff
@@ -0,0 +1,14 @@
+import { useEventListener } from '@chakra-ui/react'
+
+const useUndoShortcut = (canUndo: boolean, undo: () => void) => {
+  const isUndoShortcut = (event: KeyboardEvent) =>
+    (event.metaKey || event.ctrlKey) && event.key === 'z'
+
+    useEventListener('keydown', (event: KeyboardEvent) => {
+      if (isUndoShortcut(event) && canUndo) {
+        undo()
+      }
+    }, null, { passive: false })
+}
+
+export default useUndoShortcut
```

Comment:
Can you not export as default? I tend to do only named exports in the project

```ts
export const useUndoShortcut =...
```

I'll add a eslint rule to enforce this 👍

Reply:
refactored